skyfield/documentation/positions.rst

```diff
@@ -39,7 +39,8 @@ together with all of the attributes and methods that they support:
 
     Apparent position only
      │
-     └── `altaz(…) <api.html#Position.altaz>`_            →   alt, az, distance
+     └── `altaz(…) <api.html#Position.altaz>`_                 →   alt, az, distance
+     └── `over_location(…) <api.html#Apparent.over_location>`_ →   topos
```

Comment:
Having no background in astronomy whatsoever, I chose a name that is more intuitive for people like me. While I like to keep things as dumb as possible, the primary audience of Skyfield probably knows about things like this and we should cater to them.